src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -318,7 +328,31 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
     }
   }
 
-  public onMouseDown(event: MouseEvent): void {
+  public onHeaderGrabHandleMouseDown(event: MouseEvent): void {
+    this.onResizeHandleMouseDown(event);
+  }
+
+  public onHeaderGrabHandleKeyDown(event: KeyboardEvent): void {
+    switch (event.key.toLowerCase()) {
+      case 'arrowleft':
+        if (this.flyoutWidth < this.config.maxWidth) {
+          this.flyoutWidth = Math.min(this.flyoutWidth + this.widthStep, this.config.maxWidth);
+        }
+        break;
+
+      case 'arrowright':
```

Comment:
IE 11 uses `Left` and `Right` (instead of `ArrowLeft` and `ArrowRight`).

Reply:
Good catch! Fixed.

Reply:
The value of `event.key` is in Pascal case: `ArrowLeft`, `Left`, etc.